doc/source/user_guide/style.ipynb

```diff
@@ -1241,4 +1241,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 1
-}
+}
```

Comment:
can you revert this

Reply:
i've no real experience with selected git revert, doesn't it just get squashed?

Reply:
`git checkout upstream/master doc/source/user_guide/style.ipynb` and commit the result